Plugins/GitUIPluginInterfaces/BuildServerIntegration/IBuildServerAdapter.cs

```diff
@@ -15,5 +15,14 @@ public interface IBuildServerAdapter : IDisposable
         IObservable<BuildInfo> GetFinishedBuildsSince(IScheduler scheduler, DateTime? sinceDate = null);
 
         IObservable<BuildInfo> GetRunningBuilds(IScheduler scheduler);
+
+        /// <summary>
+        /// Called when the current repository is closed
+        /// </summary>
+        void RepositoryClosed()
```

Comment:
Do you think we could align with the .NET API semantics?
```suggestion
        void OnRepositoryChanged()
```